catalog/.env.example

```diff
@@ -26,21 +26,6 @@ REGISTRY_URL=http://flask:5000
 # staging registry instance
 #REGISTRY_URL=https://stage-registry.quiltdata.com
 
-# OAuth config
-# ============
-
-# local
-USER_API=http://auth:5002/accounts/api-root
-SIGN_OUT_URL=http://auth:5002/accounts/logout?next=%2F
-
-# github auth
-#USER_API=https://api.github.com/user
-#SIGN_OUT_URL=/
-
-# staging auth server
-#USER_API=https://stage-auth.quiltdata.com/accounts/api-root
-#SIGN_OUT_URL=https://stage-auth.quiltdata.com/accounts/logout?next=%2F
-
```

Comment:
I'm not clear why we dropped the above; are they no longer in static/config.js?

Reply:
The auth server is gone, so there isn't anything to configure?

Reply:
> The auth server is gone, so there isn't anything to configure?

exactly